### ISSUE TITLE
undefined method `workflow' for class

### ISSUE THREAD

**Issue by ridiculous** [geekq/workflow-activerecord#7]
Hi, I'm getting an undefined method error just trying to mount the module on a basic application. I'm able to reproduce it on this repo as well. I'm aware the tests pass for Ruby 2.6 and Rails 5, and I am perplexed as to why. Maybe it's something on my machine. 

### Steps to reproduce
Start a console from this projects root
```bash
irb -I./lib -r workflow_activerecord -r active_record
```
Create a model:
```ruby
class TestIssue < ActiveRecord::Base
  include WorkflowActiverecord
end
```
It blows up with:
```bash
8: from (irb):2:in `include'
7: from /workflow-activerecord/lib/workflow_activerecord.rb:9:in `included'
6: from /workflow-activerecord/lib/workflow_activerecord.rb:9:in `include'
5: from /workflow-activerecord/lib/workflow_activerecord/adapters/active_record.rb:6:in `included'
4: from /workflow-activerecord/lib/workflow_activerecord/adapters/active_record.rb:6:in `extend'
3: from /workflow-activerecord/lib/workflow_activerecord/adapters/active_record.rb:47:in `extended'
2: from /workflow-activerecord/lib/workflow_activerecord/adapters/active_record.rb:48:in `singleton class'
1: from /workflow-activerecord/lib/workflow_activerecord/adapters/active_record.rb:48:in `alias_method'
NameError (undefined method `workflow' for class `#<Class: TestIssue (call 'TestIssue.connection' to establish a connection)>')
```
Note: I can run the tests locally and they pass fine. But I can't actually use it. Same error results when including this gem in my project. This isn't blocking me, as I can use the Workflow 2.0 gem, cause I don't use scopes and can define the interface methods this gem provides on my model directly. Wanted to share anyway in case it helps.

**Comment by geekq** [comment]
If I do the "Steps to reproduce", I get the same message. Once I add `-r workflow` the error does not appear anymore. Will prepare a fix...

**Comment by geekq** [comment]
Fixed in 4.1.5

**Comment by ridiculous** [comment]
Confimed the issue is fixed. Thanks 👍
